go/cmd/dolt/commands/status.go

```diff
@@ -43,19 +43,24 @@ func Status(ctx context.Context, commandStr string, args []string, dEnv *env.Dol
 	help, _ := cli.HelpAndUsagePrinters(commandStr, statusShortDesc, statusLongDesc, statusSynopsis, ap)
 	cli.ParseArgs(ap, args, help)
 
-	stagedDiffs, notStagedDiffs, err := actions.GetTableDiffs(ctx, dEnv)
+	stagedTblDiffs, notStagedTblDiffs, err := actions.GetTableDiffs(ctx, dEnv)
 
 	if err != nil {
 		panic(err) // fix
 	}
-
-	workingInConflict, _, _, err := actions.GetTablesInConflict(ctx, dEnv)
+	workingTblsInConflict, _, _, err := actions.GetTablesInConflict(ctx, dEnv)
 
 	if err != nil {
 		panic(err) // fix
 	}
 
-	printStatus(dEnv, stagedDiffs, notStagedDiffs, workingInConflict)
+	notStagedDcDiffs, err := actions.GetDocDiffs(ctx, dEnv)
```

Comment:
feels like shortening doc to dc isn't worth the loss in clarity of the code

Reply:
Good call, just referring to them by docs/Docs now: https://github.com/liquidata-inc/dolt/pull/222/commits/160188eb46eee8bc1331f7b69453f8d3d2033286